### ISSUE TITLE
Closing unsaved tabs doesn't prompt for saving (even with the prompt setting on)

### ISSUE THREAD

**Issue by Joey79100** [HeidiSQL/HeidiSQL#741]
## Steps to reproduce this issue

1. Make sure the "Prompt to save modified files on tab close" is checked in the settings.
2. Write something in any open tab (it can be the 1st one or any other);
3. Close it, either by double-clicking it or clicking the X;
4. The tab is closed without any prompt.

## Current behavior

The tab is closed as soon as you try to close the tab, without any prompt.

## Expected behavior

When trying to close a tab that is modified and not saved, a prompt should ask if you want to save, not save, or cancel.

## Environment

* **HeidiSQL version:** 10.2.0.5681
  
* **Operating system:**   Windows 10 Education N version 1809

**Comment by ansgarbecker** [comment]
I am prompted, with the same Heidi version and settings:
![grafik](https://user-images.githubusercontent.com/7986591/63585862-60253080-c5a0-11e9-993a-99a510d507ac.png)

![grafik](https://user-images.githubusercontent.com/7986591/63585903-703d1000-c5a0-11e9-8dda-1a7f96d2a9b3.png)

**Comment by Joey79100** [comment]
Oh damn, my bad. The option was actually off, because I confused it with another one.

See I was reading [this issue](https://github.com/HeidiSQL/HeidiSQL/issues/231), and I saw this image:
![](https://user-images.githubusercontent.com/7986591/38894935-f5ead64c-428e-11e8-99ae-1c7bd0cb2e91.png)
The option was there, in the SQL tab, but in current version it has been moved and replaced by "Show the window of query warnings"... except I'm using the French version and I thought this was still the same option, just a bad translation. Turns out it was actually something completely different and I didn't look further. Sorry about this.
